### ISSUE TITLE
multchoicebox not allowing multiple selections

### ISSUE THREAD

**Issue by justinpoiroux** [robertlugg/easygui#68]
It is acting just like the choicebox function.

from easygui import *
print multchoicebox("test", "test", [1,2,3])

Literally only allows me to select one item.

**Comment by awyea** [comment]
I'm having the same problem. It shows a choicebox instead of a multchoicebox

**Comment by justinpoiroux** [comment]
Right. Same here. Very annoying.

**Comment by awyea** [comment]
I think I found the problem. In derived_boxes.py, there is a global variable __choiceboxMultipleSelect that is set to 1 for a multchoicebox. For some reason this is getting reset. I don't know enough about python to figure out why this is happening, so I just added an parameter to the __choicebox method in base_boxes.py and set that to 1. I then found all references to __choiceboxMultipleSelect in base_boxes.py and changed it to the parameter. Seems to be working for me.
